docs/topics/forms/formsets.txt

```diff
@@ -287,12 +287,20 @@ sure you understand what they do before doing so.
 a form instance with a prefix of ``__prefix__`` for easier use in dynamic
 forms with JavaScript.
 
+.. _formsets-error-messages:
+
 ``error_messages``
 ------------------
 
 The ``error_messages`` argument lets you override the default messages that the
 formset will raise. Pass in a dictionary with keys matching the error messages
-you want to override. For example, here is the default error message when the
+you want to override. Error message keys include ``'too_few_forms'``,
+``'too_many_forms'``, and ``'missing_management_form'``. To format
+``'too_few_forms'`` and ``'too_many_forms'`` error messages they must
+contain ``%(num)d``, which will be replaced with ``min_num`` and ``max_num``,
+respectively.
```

Comment:
```suggestion
``'too_many_forms'``, and ``'missing_management_form'``. The
``'too_few_forms'`` and ``'too_many_forms'`` error messages may contain
``%(num)s``, which will be replaced with ``min_num`` and ``max_num``,
respectively.
```

Reply:
As I said [here](https://github.com/django/django/pull/15671#issuecomment-1122000794) we may have to rephrase this. What I came up is:

```suggestion
``'too_many_forms'``, and ``'missing_management_form'``. The
``'too_few_forms'`` and ``'too_many_forms'`` error messages can be formatted if
they contain ``%(num)d``, which will be replaced with ``min_num`` and ``max_num``,
respectively.
```

Reply:
Strings can be formatted even if they don't contain `%(num)d`, e.g.
```
>>> gettext_lazy("Custom message without num.") % {"num": 1}
```
works perfectly fine. I'd keep `may contain`.